### DISCUSSION TITLE
Wallet app exits

### DISCUSSION THREAD

**Discussion by aa1135435558** [Chia-Network/chia-blockchain#1926] [General]
The Chia wallet application is abnormally exiting and drawing is in progress .I checked the background process and found that Chia was still running, and SSD was also reading and writing. At this time, I restarted the wallet, but I could not see the drawing progress. Do I need to kill the background process and start again? Or wait for the background process to finish drawing?
![6FCADB6A-8954-4413-BBD2-61A0871D3863](https://user-images.githubusercontent.com/67729044/114859463-5d361e00-9e1d-11eb-8599-ebb755e3ed92.jpeg)

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
